gokv/go.mod

```diff
@@ -0,0 +1,13 @@
+module github.com/philippgille/gokv/gokv
```

Comment:
What do you think about moving the module to `github.com/philippgille/gokv/cmd/gokv`? And the `gen` module could live under the cmd path besides the gokv path.

That would especially make the tagging / releasing nicer. When tagging/releasing submodules, the current way the tag would probably have to be `gokv/v0.1.0`, which could be confused with the top level module that only contains the interface. `cmd/gokv/v0.1.0` makes the purpose of the module more clear.